tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -49,6 +49,12 @@ public static function prepare($scope)
 
         self::$kernel = new AppKernel('test', true);
         self::$kernel->boot();
+
+        global $kernel;
+        $kernel = self::$kernel;
+
+        $employeeId = \Employee::employeeExists('test@prestashop.com');
+        \Context::getContext()->employee = new \Employee($employeeId);
```

Comment:
I guess hardcoding 1 as paremeter would be enough for the tests, don't you think?

Reply:
Why not